xml/Microsoft.Extensions.Localization/StringLocalizer`1.xml

```diff
@@ -91,9 +91,9 @@
         <Parameter Name="includeParentCultures" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="includeParentCultures">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="includeParentCultures">A <see cref="T:System.Boolean" /> indicating whether to include strings from parent cultures.</param>
+        <summary>Gets all string resources.</summary>
+        <returns>The strings.</returns>
```

Comment:
```suggestion
        <param name="includeParentCultures"><see langword="true" /> to include strings from parent cultures; <see langword="false" /> otherwise.</param>
        <summary>Gets all the resource strings.</summary>
        <returns>An enumeration that contains all the resource strings.</returns>
```